index.html

```diff
@@ -3,14 +3,15 @@
   <head>
     <meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
 <!--
-== OpenJSCAD.org, Copyright (c) 2013-2016 by Rene K. Mueller <spiritdude@gmail.com>, Licensed under MIT License ==
+== OpenJSCAD.org, Copyright (c) 2013-2017 by Rene K. Mueller <spiritdude@gmail.com>, Licensed under MIT License ==
    with some code from OpenJsCad processfile.html by Joost Nieuwenhuijse
    in conjunction with csg.js, openjscad.js, lightgl.js by various authors (see them listed in the individual files)
 
 Purpose:
    More modern interface for OpenJsCad as published at http://joostn.github.com/OpenJsCad/
 
 History:
+  2017/04/29: 0.6.0: reworked to use new bar library; index.js
```

Comment:
what is 'Bar' ??? honestly this is not very clear , and there is already information in the README, changelog etc, let us just remove this redundant information from individual files, it add a huge amount of extra work for little to no benefit :)